packages/util/src/logger/cloud-sdk-logger.ts

```diff
@@ -16,12 +16,36 @@ const container = new Container();
 const exceptionTransport = new transports.Console();
 const customLogLevels = {};
 const DEFAULT_LOGGER__MESSAGE_CONTEXT = '__DEFAULT_LOGGER__MESSAGE_CONTEXT';
+let silent = false;
 
 const moduleLogger = createLogger({
   package: 'util',
   messageContext: 'cloud-sdk-logger'
 });
 
+function toggleMuteLoggers(silence: boolean) {
```

Comment:
Minor: Method is not exported so not so important, but I am a bit confused `silent` and `silence` both a boolean muting so why not use the same name.

Reply:
Because silent is a local variable 😕 I just didn't have a better idea.

Reply:
ah shadow, consider using a namespace for the private scope above.